crates/pop-parachains/src/call/mod.rs

```diff
@@ -36,6 +36,14 @@ pub async fn construct_extrinsic(
 	Ok(subxt::dynamic::tx(pallet_name, extrinsic_name, parsed_args))
 }
 
+/// Constructs a Sudo extrinsic.
+///
+/// # Arguments
+/// * `tx`: The transaction to be executed with sudo privileges.
```

Comment:
Terminology is off, 'sudo privileges' doesnt sound right. See command param description and above comment for more info.

Reply:
Changed to `* `tx`: The transaction payload representing the function call to be dispatched with `Root` privileges.`